Lib/defcon/objects/font.py

```diff
@@ -884,6 +893,8 @@ def _get_identifiers(self):
     # object as dirty.
 
     def _get_guidelines(self):
+        if self._info is None:
+            self.info
```

Comment:
Thanks Adrien, I tagged [0.3.2](https://github.com/typesupply/defcon/releases/tag/0.3.2) with this.